companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -1717,18 +1728,16 @@ class TelemetryCurrentSourceConversionTable: public ConversionTable
     }
 };
 
-class FrskyField: public StructField {
+class VarioField: public StructField {
```

Comment:
I am not sure it is easier to read now, as we mix telemetry screens (218) in a structure which should deal with vario.
Instead we could create a class Frsky218Field and another class VarioField (without screens inside).

Reply:
ok, will do!